doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +163,98 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
 
    The response should list three running nodes.
 
+.. _amazon-aws:
+
+Setting up Kubernetes on Amazon Web Services (AWS)
+--------------------------------------------------
+
+AWS does not have native support for Kubernetes, however there are
+many organizations that have put together their own solutions and
+guides for setting up Kubernetes on AWS. We like the `guide put
+together by Heptio <https://s3.amazonaws.com/quickstart-reference/heptio/latest/doc/heptio-kubernetes-on-the-aws-cloud.pdf>`_,
+and recommend using this for setting up your stack.
+
+1. Follow Step 1 of the Heptio guide. This sets up your Amazon account with
+   the credentials needed to run Kubernetes.
+
+   .. note::
+
+      Make sure that you keep the file downloaded when you create the SSH
+      key. This will be needed later to allow ``kubectl`` to interact with
+      your Kubernetes cluster.
+
+   .. note::
+
+      You may find it helpful to "pin" the services we'll be using to your AWS
+      navbar. This makes it easier to navigate in subsequent sessions.
+      Click the "pin" icon at the top, then drag ``CloudFormation`` and
+      ``EC2`` into your navbar.
+
+2. Follow Step 2 of the Heptio guide. In this section, Heptio allows you
+   to click one of two buttons, each of which run several commands on AWS
+   that set up a "template" of resources we need for Kubernetes.
+   We recommend ``Option 1``, which will create a new set of resources
+   on AWS to run Kubernetes.
+
+   After clicking the link you'll be taken to an AWS page with a field already
+   chosen under "Choose a template". Simply hit "Next". On the following
+   page you'll need to fill in some information. Here are the required fields:
+
+   * ``Stack Name`` can be anything you like.
+   * ``Availability Zone`` is related to the location of the AWS
+     resources. Choose any.
+   * ``Admin Ingress Location`` defines the locations from which you
+     can access this cluster as an administrator. Just enter ``0.0.0.0/0``
+     for the most permissive approach.
+   * ``SSH Key`` is a dropdown list of keys attached to your account.
+     The one you created in Step 1 should be listed here. This will allow
+     you to SSH into the machines if you desire. This will depend on
+     the ``Instance Type`` that you choose. Remember that the most
+     common bottleneck is RAM.
```

Comment:
This is a bit confusing. I would remove ~~This will depend...is RAM.~~ or reword.